about.md

```diff
@@ -6,9 +6,11 @@ title: About
 We meet monthly and quarterly to discuss and learn about the Linux family of operating systems and other free-and-open source software.
 
 ## History
+###Fort Wayne LUG 0.0: Genesis
```

Comment:
Should have an empty line after the History heading, to preserve pre-change file structure.

Markdown headings will fail to render without a space between the pound symbols and the heading title.